README.md

```diff
@@ -11,7 +11,17 @@
 
 A collection of various means for loading various different corpora used in NLP.
 
-
+## Installation
+As per the standard [Julia package installation](https://julialang.github.io/Pkg.jl/dev/managing-packages/#Adding-registered-packages-1):
+
+```julia
+julia> Pkg.add("CorpusLoaders")
+```
+Also, in the Pkg REPL, the package can be added with the add command, as: 
+```julia
+pkg> add CorpusLoaders
+```
+**Note**: Only linux and OS X are supported.
```

Comment:
really? why is that?

Reply:
I saw that all of the JuliaText packages had the Installation instructions in their README.md. However this one didn't, so I thought of adding it.

Reply:
Why does this say not compatible with windows?

Reply:
Yes, I saw that. 
It was written in a few other packages. 
I'm sorry but I didn't check before the commit. Should I remove the **Note** and commit?